storybook/src/elements/nuxeo-icons/nuxeo-icons.stories.js

```diff
@@ -0,0 +1,51 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-card.js';
+import { IronMeta } from '@polymer/iron-meta/iron-meta.js';
+
+const items = new IronMeta({ type: 'iconset' }).list;
+items.forEach((item) => {
+  item.icons = item.getIconNames();
+});
```

Comment:
Since we only care about `name` and `icons`, we may want to simplify the object.
WDYT about using something like this:
`const items = new IronMeta({ type: 'iconset' }).list.map((item) => {
  return { name: item.name, icons: item.getIconNames() };
});`

Do you think it fulfils our needs here?

Reply:
changed